workspaces/mi/mi-extension/src/rpc-managers/agent-mode/rpc-manager.ts

```
@@ -0,0 +1,1380 @@
+    '.idea',
+    '.settings',
+    'dist',
+    'build',
+    'out',
+    'target',
+]);
+
+// Session management types (will be imported from @wso2/mi-core after build)
+export interface ListSessionsRequest {
+    // Empty - uses project from context
+}
```

Biome (2.3.14)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)